src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,70 @@
+package codeu.chat.util;
+
+import java.util.LinkedList;
+
+public final class Tokenizer {
```

Comment:
This file could use some comments. (Sadly, the original project code didn't set a great example in this respect.) At a bare minimum, there should be Javadoc for the class and each public method (including the constructor).